test/v1/connectors/ConnectorSpec.scala

```diff
@@ -154,6 +188,7 @@ trait ConnectorSpec extends UnitSpec with Status with MimeTypes with HeaderNames
     MockAppConfig.tysIfsEnvironment returns "TYS-IFS-environment"
     MockAppConfig.tysIfsEnvironmentHeaders returns Some(allowedDownstreamHeaders)
 
-    MockAppConfig.featureSwitches returns Configuration("tys-api.enabled" -> true)
+    MockAppConfig.featureSwitches returns Configuration("tys-api.enabled" -> true) anyNumberOfTimes()
```

Comment:
Just out of interest, what added value does `anyNumberOfTimes` give here?

Reply:
Without it, the stub expects to be called exactly once, which causes connector specs for ones which don't check for TYS to fail.